cmd/rook/object/connection.go

```diff
@@ -72,13 +80,15 @@ func getConnectionInfoEntry(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func getConnectionInfo(format string, c client.RookRestClient) (string, error) {
+func getConnectionInfo(c client.RookRestClient, userID, format string) (string, error) {
+	// TODO: support other types of connection info to object store (such as swift)
+	// https://github.com/rook/rook/issues/255
```

Comment:
I'm happy to remove this but it wasn't something I added. It got moved in a refactoring.